zerver/lib/test_runner.py

```diff
@@ -104,6 +106,18 @@ def run_test(test: TestCase, result: TestResult) -> bool:
 
     test_name = full_test_name(test)
 
+    # Write each test being run to a file classified by its PID.
+    # This allows reference to the set of tests ran in a failed worker.
+    fn = "var/test-backend-%s.json" % (os.getpid(),)
```

Comment:
I think these paths names are not unique enough.  We should go with `var/test-backend/test-ordering/log-%s.json` as the path (probably with some `os.makedirs(..., exist_ok=True)`.  (We have an open issue for moving everything `test-backend` generates directly to things under `var/test-backend/`.)

Reply:
I'm also not entirely sure why we need these files at all -- can't we just store those in a variable and then send them back to the parent process the same way we tell it whether we succeeded?  

One might need to read some of the Django code for `ParallelTestSuite` to see if that's possible, but it would be nice if we could.